### PR TITLE
Feature/129 treat certain punctuation characters as regular characters

### LINKED ISSUE #129: Treat certain punctuation characters as regular characters
## Description
Punctuation characters have a longer delay than non punctuation characters. HOWEVER sometimes punctuation should not have a delay. Find a way to make it so characters can be ignored but only in certain cases e.g., a period "." should have a longer delay, but the same period in "Mr. Boy" should not have a longer delay.

Wait for #103 to be accepted before starting this feature.

## Acceptance Criteria

This story will be finished when punctuation has been demonstrated to have a long delay and a short delay in one sentence.

### PR DESCRIPTION
**Pull request by PhoebeMitchell**
## Summary

Adds functionality to allow punctuation to be marked as regular characters without extra delay. Example: to treat a full stop as a regular character write: `<link=character>.<link>` or `{char(".")}` if including Macros.ink

## Testing instructions

Run PunctuationWithRegularDelayDemo and watch the full stops appearing at a normal rate.

## Additional information

- `[ ]` Changes UI
- `[x]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  
closes #129

### CODE REVIEW

File: unity-ggjj/Assets/Resources/InkDialogueScripts/TestScripts/New Ink.json [hunk 1]
```diff
@@ -0,0 +1 @@
+﻿{"inkVersion":20,"root":[[["done",{"#f":5,"#n":"g-0"}],null],"done",{"#f":1}],"listDefs":{}}
```
Comment (ViMaSter) [inline unity-ggjj/Assets/Resources/InkDialogueScripts/TestScripts/New Ink.json]: I think this file got added by accident? I don't see any references to it and no `.ink`-file that matches.

**Review by ViMaSter — DISMISSED** [review]
Simple and straight to the point; awesome job! 👍 (One stray file, it seems?)

File: unity-ggjj/Assets/Resources/InkDialogueScripts/TestScripts/New Ink.json (hunk 1 above)
Comment (PhoebeMitchell) [reply unity-ggjj/Assets/Resources/InkDialogueScripts/TestScripts/New Ink.json]: Whoops hehe x) Deleted

**Review by ViMaSter — APPROVED** [review]

**Review by lyndsiWilliams — APPROVED** [review]
Looks good, works good, tests good! ✅

**Pull request merged**